psamm/datasource/sbml.py

```diff
@@ -692,6 +692,23 @@ def _create_unique_id(self, in_dict, orig_id):
                 suffix += 1
         return orig_id
 
+    def _add_flux_bounds(self, r_id, model, flux_limits, equation):
+        if r_id not in flux_limits or flux_limits[r_id][0] is None:
+            if equation.direction == Direction.Forward:
+                lower_str = text_type(0)
+            else:
+                lower_str = text_type(-model.get_default_flux_limit())
+        else:
+            lower_str = text_type(flux_limits[r_id][0])
+        if r_id not in flux_limits or flux_limits[r_id][1] is None:
+            if equation.direction == Direction.Reverse:
+                upper_str = text_type(0)
+            else:
+                upper_str = text_type(model.get_default_flux_limit())
+        else:
+            upper_str = text_type(flux_limits[r_id][1])
+        return lower_str, upper_str
+
```

Comment:
Now that I look at this again I start to think that this is incorrect according to the FBCv2 spec. The `lowerFluxBound` and `upperFluxBound` should not contain the actual values but should instead contain references to the parameters defined below. Can you comment on that?